database/queries/manual_queries.sql

```diff
@@ -5,7 +5,7 @@
 -- issue - Hotfix 161.
 update websites set noc='remove' where site_url = 'alanstorm.com';
 
--- Based on other permissions (approcal.txt) as of 10th November 2016.
+-- Based on other permissions (approval.txt) as of 10th November 2016.
```

Comment:
Yes I do know this workflow but actually I was "shying" from creating an issue over such a small typo. Thank you for guiding me @kapilsharma . I'll strictly follow this workflow in the future.